<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Client">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>681</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations Client</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>173</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_PushButton_Infos">
     <item>
      <widget class="QPushButton" name="pushButton_Name">
       <property name="text">
        <string>Nom de l'ordinateur distant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_User">
       <property name="text">
        <string>Nom de l'utilisateur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Archi">
       <property name="text">
        <string>Architecture processeur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OS">
       <property name="text">
        <string>OS de l'ordinateur</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_LineEdit_Infos">
     <item>
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_User"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Archi"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_OS"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Serveur">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur</string>
   </property>
   <widget class="QTextEdit" name="textEdit_Etat">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_PushButton_Connexion">
     <item>
      <widget class="QPushButton" name="pushButton_Quitter">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Connexion">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Connexion</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_Etat">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>État de la connexion</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_Infos">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Label_Infos">
       <item>
        <widget class="QLabel" name="label_IP">
         <property name="text">
          <string>Adresse IP du serveur :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Port">
         <property name="text">
          <string>Numéro du port du serveur :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_LineEdit_Infos_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_IP">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_Port">
         <property name="minimum">
          <number>8888</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
